src/Adapter/Order/CommandHandler/UpdateProductInOrderHandler.php

```diff
@@ -58,54 +60,65 @@ final class UpdateProductInOrderHandler extends AbstractOrderHandler implements
      */
     private $orderProductQuantityUpdater;
 
+    /**
+     * @var ContextStateManager
+     */
+    private $contextStateManager;
+
     /**
      * UpdateProductInOrderHandler constructor.
      *
      * @param OrderProductQuantityUpdater $orderProductQuantityUpdater
+     * @param ContextStateManager $contextStateManager
      */
     public function __construct(
-        OrderProductQuantityUpdater $orderProductQuantityUpdater
+        OrderProductQuantityUpdater $orderProductQuantityUpdater,
+        ContextStateManager $contextStateManager
```

Comment:
Isn't this a BC break ?
We can avoid it by allowing nullable argument 😉

Reply:
Fixed.